uint32 jointOwner
Entity meEntity
